### PR TITLE
add shoestring v1.0.5 w/ npm auto-update

### PR DESCRIPTION
**Pull request by maruilian11**
@Amomo @Piicksarn could u help me check this PR for #9134 ? thank you!
**though git repo has more version, dist on git repo appear since v1.0.3 released.** 

**git repo url:** https://github.com/filamentgroup/shoestring
**Watch 30
  Star 430
 Fork 15**

**npm package url:** https://www.npmjs.com/package/shoestring
**49 downloads in the last day
244 downloads in the last week
1,232 downloads in the last month**

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR 
 * [x] The lib has notable popularity  
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry 
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update) 
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/shoestring/package.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/filamentgroup/shoestring.git"
+  },
+  "author": {
+    "name": "Filament Group",
+    "email": "thegroup@filamentgroup.com",
+    "url": "filamentgroup.com"
+  },
+  "license": "MIT",
```
Comment (PeterDaveHello) [inline ajax/libs/shoestring/package.json]: https://github.com/filamentgroup/shoestring/blob/v1.0.5/dist/shoestring.js#L3

File: ajax/libs/shoestring/package.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    "DOM-ready",
+    "HTML"
+  ],
+  "homepage": "https://filamentgroup.github.io/shoestring/dist/docs/",
+  "npmName": "shoestring",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "!(*-dev*)"
```
Comment (PeterDaveHello) [inline ajax/libs/shoestring/package.json]: I didn't see a reason to drop `shoestring-dev.js`

**Comment by maruilian11** [comment]
@Amomo @Piicksarn could you help me check this PR?
I have modify the license and add GPL-2.0 for it.
After reading [features](https://github.com/filamentgroup/shoestring#features), I thought we should add `-dev` files, too.

**Comment by PeterDaveHello** [comment]
Don't you need to confirm with its author first?

**Pull request merged**